### ISSUE TITLE
Problems with opening large files without linebreaks

### ISSUE THREAD

**Issue by th-joerger** [notepad-plus-plus/notepad-plus-plus#10407]
### Description of the Issue
Notepad++ crashes when opening large files without linebreaks, e. g. Firefox profiler dumps.

### Steps to Reproduce the Issue

1. Record a large Firefox profiler output (F12 -> record) until the buffer is full.
2. Stop and save the JSON.
3. Open the JSON.

### Expected Behavior
Notepad++ should not crash.

### Actual Behavior
Notepad++ stops reacting and does not display the file.

### Debug Information
Notepad++ v8.1.3   (32-bit)
Build time : Aug 10 2021 - 00:32:53
Path : C:\Program Files (x86)\Notepad++\notepad++.exe
Command Line : 
Admin mode : OFF
Local Conf mode : OFF
Cloud Config : OFF
OS Name : Windows 10 Enterprise (64-bit) 
OS Version : 2009
OS Build : 19042.870
Current ANSI codepage : 1252
Plugins : ComparePlugin.dll DSpellCheck.dll mimeTools.dll NppConverter.dll NppExport.dll PythonScript.dll

**Comment by felix-22** [comment]
Related / duplicate of https://github.com/notepad-plus-plus/notepad-plus-plus/issues/5267. 

(If you agree / confirm that this is a duplicate, could you close the issue and comment in the earlier thread.)

**Comment by ArkadiuszMichalski** [comment]
File for test (30MB after unpack): [Firefox 2021-08-20 15.20 profile.json.gz](https://github.com/notepad-plus-plus/notepad-plus-plus/files/7021783/Firefox.2021-08-20.15.20.profile.json.gz)

Notepad ++ actually crashes. I checked SciTE and Notepad2 and there is no problem with this file. Maybe they improved something in the new Scintilla, or the problem is with Notepad++.

**Comment by felix-22** [comment]
Basically it does not  hang/crash for me, neither with **Word wrap** enabled nor with disabled, it takes several seconds to load, and then I can scroll up/down and move cursor left/right/up/down, <== UI is a bit jerky while doing such actions, but still usable, however while doing other similiar actions like trying to type or remove a char it is becoming very laggy like in https://github.com/notepad-plus-plus/notepad-plus-plus/issues/5267, almost unusable.

However I can reproduce the issue with **Word wrap** enabled, after I minimize N++ to the Windows taksbar and then unminimize back, N++ 's GUI becomes faded, and N++ freezes / does not respond, and I have to kill it's process via Windows Taks Manager window.

(tested in 7.9.5 32bit and 8.1.3 32bit)

**Comment by th-joerger** [comment]
> 
> 
> Related / duplicate of #5267.
> 
> (If you agree / confirm that your issue is a duplicate, could you close your issue and comment in the earlier thread.)

I do not agree, because in one case it starts lagging and in this case, it hard crashes.

**Comment by felix-22** [comment]
Yeah, it seems that the two issues: lag and crash are separate ones.
